portfolio.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Site</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <header class="navtools_container">
+      <h2 class="logo">Welcome</h2>
+ 
+      <nav class="navbar">
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </header>
+
+    <main class="main_container">
+      <h1 class="item1">Hey there. I'm Hamza</h1>
+
+      <h1 class="item2">I'm a software developer</h1>
+
+      <P class="item3">
+        I can help you build a product, feature or website 
+        Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.
+      </P>
+
+    <section class="icon_container">
+        <a href="https://www.twitter.com/"><img src="twitter.png" alt="Twitter icon"></a>
+        <a href="https://www.linkedin.com/"><img src="Linkedin icon.png" alt="linkedin icon"></a>
+        <a href="https://medium.com/"><img src="medium.png" alt="medium icon"></a>
+        <a href="https://github.com/"><img src="github.png" alt="github icon"></a>
+        <a href="https://www.angellist.com/"><img src="angellist.png" alt="angellist icon"></a>  
+    </section>
```

Comment:
- Please as required by the previous reviewer make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). instead of using the tag  section  :+1:  

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

```
eg  
<ul class="headline-img">
      <li><a href="#"><img class="t-social-media" src="img/twitter.png" alt="twitter"></a></li>
      <li><a href="#"><img class="l-social-media" src="img/linkedin_icon.png" alt="linkedIn"></a></li>
      <li><a href="#"><img class="m-social-media" src="img/medium.png" alt="medium"></a></li>
      <li><a href="#"><img class="g-social-media" src="img/github.png" alt="github"></a></li>
      <li><a href="#"><img class="v-social-media" src="img/vec.png" alt="vector"></a></li>
    </ul>
```